website/content/docs/sync/azurekv.mdx

```diff
@@ -5,3 +5,89 @@ description: The Azure Key Vault destination syncs secrets from Vault to Azure.
 ---
 
 # Azure Key Vault
+
+The Azure Key Vault destination enables Vault to sync and unsync secrets of your choosing into
+an external Azure account. When configured, Vault will actively maintain the state of each externally-synced
+secret in realtime. This includes sending new secrets, updating existing secret values, and removing
+secrets when they either get dissociated from the destination or deleted from Vault.
+
+## Setup
+
+A service principal with a client id and client secret will be needed to configure Azure Key Vault as a
+sync destination. This [guide](https://learn.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal)
+will you through creating the service principal.
+
+Once the service principal is created, the next step is to
+[grant the service principal](https://learn.microsoft.com/en-us/azure/key-vault/general/rbac-guide?tabs=azure-cli)
+access to Azure Key Vault. We recommend using the "Key Vault Secrets Officer" built-in role,
+which gives sufficient access to manage secrets.
```

Comment:
Should we still have a authentication or permissions section to list the minimum requires privileges even if we recommend a built-in role?